worker/include/RTC/IceServer.hpp

```diff
@@ -61,15 +61,15 @@ namespace RTC
 		{
 			return this->selectedTuple;
 		}
-		void SetUsernameFragment(const std::string& usernameFragment)
+		void RestartIce(const std::string& usernameFragment, const std::string& password)
 		{
 			this->oldUsernameFragment = this->usernameFragment;
 			this->usernameFragment    = usernameFragment;
-		}
-		void SetPassword(const std::string& password)
-		{
+
 			this->oldPassword = this->password;
 			this->password    = password;
+
```

Comment:
Please avoid this blank line.

Reply:
The blank line here is to separate different assignment operation to make it more readablem
```
void RestartIce(const std::string& usernameFragment, const std::string& password)
{
	this->oldUsernameFragment = this->usernameFragment;
	this->usernameFragment    = usernameFragment;

	this->oldPassword = this->password;
	this->password    = password;
        
        this->remoteNomination = 0;
}
```